File: Corrections/php/Neufplate/src/PriceCalculator/IncludingTaxesPriceDecorator.php
<?php

namespace App\PriceCalculator;

class IncludingTaxesPriceDecorator extends PriceDecorator
{

    public function __construct(Price $price)
    {
        parent::__construct($price);
    }

    public function calculate(): float
    {
        $this->baseValue = parent::calculate() * 1.2;
        return $this->baseValue;
    }

}

File: Corrections/java/Neufplate/src/main/java/fr/neufplate/PriceCalculator/IncludingTaxesPriceDecorator.java
package fr.neufplate.PriceCalculator;

public class IncludingTaxesPriceDecorator extends PriceDecorator {
    public IncludingTaxesPriceDecorator(Price price) {
        super(price);
    }

    @Override
    public double calculate() {
        this.baseValue = super.calculate() * 1.2d;
        return this.baseValue;
    }
}
